shared/wallets/transaction/index.js

```diff
@@ -252,16 +249,12 @@ export const TimestampLine = (props: TimestampLineProps) => {
     )
   }
   if (!props.timestamp) {
-    return (
-      <Text type="BodySmall">
-        {props.relative ? 'Pending' : "The Stellar network hasn't confirmed your transaction."}
```

Comment:
what was the relative/pending thing?

Reply:
Holdover from before we switched the timestamp format to be the same as Chat -- after doing that we can just use the same text on both screens.